### PR TITLE
WRP-3049: Update ilib dependency to latest version

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated ilib dependency to latest version '^14.15.1'.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3049

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "warning": "^4.0.3",
-    "ilib": "14.14.0"
+    "ilib": "^14.15.1"
   },
   "peerDependencies": {
     "ilib": "14.14.0 || ^14.14.0-webos1"
```
Comment (daniel-stoian-lgp) [inline package.json:60]: you mised this one :)
Take PR #1322 as reference

**Comment by daniel-stoian-lgp** [comment]
Since package-lock.json was not modified in PR #1322, I am not sure if it should be added to this PR. I will ask the next reviewer to take a decision

File: package.json (hunk 1 above)
Comment (daniel-stoian-lgp) [reply package.json:60]: "ilib": "^14.15.1 || ^14.15.1-webos1"

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Comment by hong6316** [comment]
Please update/sync `feature/WRP-3049` with the latest `develop` branch and resolve conflicts.

**Comment by andreitirlalge** [comment]
Closing this PR since ilib version is already updated on develop branch.

**Pull request closed without merging**